Sources/AWSLambdaRuntimeCore/Terminator.swift

```diff
@@ -20,11 +20,7 @@ import NIOCore
 public final class LambdaTerminator {
     fileprivate typealias Handler = (EventLoop) -> EventLoopFuture<Void>
 
-    private var storage: Storage
-
-    init() {
-        self.storage = Storage()
-    }
```

Comment:
More code cleanup. You don't need the init if all you're doing is providing a default value to the private `storage` variable